packages/server/src/adapters/standalone.ts

```diff
@@ -19,14 +19,9 @@ export function createHTTPHandler<TRouter extends AnyRouter>(
   opts: CreateHTTPHandlerOptions<TRouter>,
 ) {
   return async (req: http.IncomingMessage, res: http.ServerResponse) => {
-    // if no hostname, set a dummy one
-    const href = req.url!.startsWith('/')
```

Comment:
I'm unsure the ramifications of changing this bit?

Reply:
There are none, it's effectively the same thing, but using `URL` constructor directly.

Reply:
Cool cool :)